File: padroesCriacionais/padrao-de-projeto-abstractFactory/java/src/test/java/HardEnemiesTest.java
import abstractFactory.HardEnemies;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class HardEnemiesTest {

    @Test
    public void shouldReturnHardEnemies() {
        HardEnemies hardEnemies = new HardEnemies();
        assertEquals("gera inimigos no modo dificil", hardEnemies.makeEnemies());
    }
}

File: padroesCriacionais/padrao-de-projeto-abstractFactory/php/tests/unit/HardEnemiesTest.php
<?php

namespace Abstractfactory\tests\unit;

use Abstractfactory\Php\HardEnemies;
use PHPUnit\Framework\TestCase;

class HardEnemiesTest extends TestCase
{
    public function testShouldReturnHardEnemies(): void
    {
        $hardEnemies = new HardEnemies();
        $this->assertEquals("gera inimigos no modo dificil", $hardEnemies->makeEnemies());
    }
}